examples/react-map-gl-draw/package.json

```diff
@@ -18,7 +18,6 @@
     "@babel/plugin-proposal-class-properties": "^7.0.0",
     "@babel/plugin-proposal-export-default-from": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
-    "@babel/preset-flow": "^7.0.0",
```

Comment:
Aren't you going to need a typescript preset since you are changing the js files to ts?